ext/ed.h

```diff
@@ -27,7 +27,7 @@ class SslBox_t; // forward reference
 #endif
 
 bool SetSocketNonblocking (SOCKET);
-
+bool SetFdCloexec (int);
```

Comment:
Oh, duh - nonblocking applies only to socket, cloexec applies to any FD. Let's keep it as-is.